styles/styles.css

```diff
@@ -0,0 +1,337 @@
+@import url('https://fonts.googleapis.com/css?family=Archivo+Black|Merriweather:400,400i');
+
+.flex-container {
+    display: flex;
+    /*Need to decide on columns, rows, or other approach*/
+    /*May also need to figure out if you can do multiple grids in one page and if so whether that is best approach here*/
+}
+
+a:link {
+  text-decoration: none;
+}
+  
+a:visited {
+  text-decoration: none;
+}
+  
+#banner {
+  background-color: #093b60;
+  height: 18em;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+}
+
+a.subtitle:visited {
+  color: #093b60;
+}
+
+#letterhead {
+  display: flex;
+  flex-direction: column;
+  
+  margin-top: 3%;
+  color: #81B642;
+  font-family: Merriweather, Times, serif;
+  font-style: italic;
+  text-align: center;
+  font-size: 150%;
+}
+
+.filler {
+  width: 0%;
+  
+ 
+}
+
+#toplogo {
+  max-width: 100%;
+  height: auto;
+}
+
+#cart {
+  display: flex;
+  flex-direction: column;
+  justify-content: right;
+  margin-top: 2rem;
+   
+}
+
+.top-cart {
+  color: white;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+}
+
+
+#carticon {
+/*Size to make small; I've just shrunk it a bit for now*/
+width: 1.5em;
+
+}
+
+.checkout-button {
+  /*Green background, position beneath cart icon */
+    background-color: #81B642;
+    height: 30px;
+    width: 140px;
+    color: white;
+    font-family: "Archivo Black", Geneva, sans-serif;
+    font-weight: bold;
+    font-size: 75%;
+    line-height: 30px;
```

Comment:
do a search on Mozilla Developer's network on line-height. I'm a fan of the multiplier method for these

Reply:
Sorry, but I'm not quite catching your drift on this one. I've done the search for line-height in MDN but am not sure if if I've found what you've called the multiplier method. Is this the same as what they're describing as:
/* Unitless values: use this number multiplied
by the element's font size */
line-height: 3.5;

?

Thanks!

Reply:
Yeah, honestly....don't worry about it, if you're happy with 30px, that's fine. I don't know why I was nitpicky about it. Must have had a bee in the bonnet...